contrib/gitea-pull-checkout

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
# $1 = user:branch as per copy button on PR page
# $2 = repo name, defaults to gitea
```
or regarding the suggestion below
```suggestion
#!/bin/bash
# $1 = "user:branch" as per copy button on PR page
# $2 = PR number to keep track of branches that were created for reviewing PRs instead of from you yourself
# $2 = repo name, defaults to "gitea"
```